tensorflow/examples/tutorials/mnist/mnist_softmax_xla.py

```diff
@@ -102,6 +102,14 @@ def main(_):
       default='/tmp/tensorflow/mnist/input_data',
       help='Directory for storing input data')
   parser.add_argument(
-      '--xla', type=bool, default=True, help='Turn xla via JIT on')
+      '--no-xla',
+      dest='xla', action='store_false',
```

Comment:
I'm OK with that. I'd just like this tutorial to work as documented. :-)

It might be a while before I can get a committers agreement signed, so anyone else is welcome to submit an equivalent PR.